src/utils/validator.ts

```diff
@@ -73,5 +73,27 @@ export const validators = [
         moment(field.value).isBefore(moment().add(field[UI.MAX_DATE_OFFSET], 'd')),
         field[UI.VALIDATION_ERROR]
       )
+  },
+  {
+    name: UI.MIN_DATE_TIME_OFFSET,
+    action: (field: any) => {
+      if (field[UI.MIN_VALUE]) return actionMethod(true, 'Skipped validation')
```

Comment:
What is the string here used for?

Reply:
Since the `condition` argument value is `true` I believe the `message` argument is not used anywhere. I think of it as a self documenting  string since `field[UI.VALIDATION_ERROR]` is not appropriate.